bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/StaticImage.java

```diff
@@ -0,0 +1,61 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Lets name the file "Image.java" only

Reply:
we're already using Image object coming from wcm.day so used staticImage to give a different name